homeassistant/components/dsmr/sensor.py

```diff
@@ -349,6 +349,24 @@ class DSMRSensorEntityDescription(
         device_class=SensorDeviceClass.ENERGY,
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
+    DSMRSensorEntityDescription(
+        key="belgium_current_average_demand",
+        translation_key="current_average_demand",
+        obis_reference=obis_references.BELGIUM_CURRENT_AVERAGE_DEMAND,
+        dsmr_versions={"5B"},
+        force_update=True,
+        device_class=SensorDeviceClass.POWER,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    DSMRSensorEntityDescription(
+        key="belgium_maximum_demand_month",
+        translation_key="maximum_demand_month",
+        obis_reference=obis_references.BELGIUM_MAXIMUM_DEMAND_MONTH,
+        dsmr_versions={"5B"},
+        force_update=True,
+        device_class=SensorDeviceClass.POWER,
+        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
This is an aggregated value, and thus cannot have a state class.

Reply:
What you mean by its an aggregated value?
This value comes straight from the DSMR Telegram, and changes every second (in fact it shows how much kWh you used * 4, so you know at the end of every 15 minutes how much your peak usage was)

Reply:
Fixed